site/src/main/java/uk/nhs/digital/website/beans/ApiMaster.java

```diff
@@ -0,0 +1,69 @@
+package uk.nhs.digital.website.beans;
+
+import org.hippoecm.hst.content.beans.Node;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.content.beans.standard.HippoHtml;
+
+import org.onehippo.cms7.essentials.dashboard.annotations.HippoEssentialsGenerated;
+
+import java.util.List;
+
+@HippoEssentialsGenerated(internalName = "website:apimaster")
+@Node(jcrType = "website:apimaster")
+public class ApiMaster extends BaseDocument {
```

Comment:
There is a file called CommonFieldsBean which a lot of doc types extend.  Then could remove getSeosummary, getSummary and getSummary from this file.

Reply:
Thanks @ngta1. 
 I addressed the change you mentioned. Also renamed 'apimaster' to 'API master' for CMS UI in repository-data/application/src/main/resources/hcm-config/configuration/translations/templates.yaml

Additionally, branch is rebased to the tip of 'master'

Please have a look and let me know your thoughts. 
Thanks you,
Damian

Reply:
Great thanks Damian.